pdu.go

```diff
@@ -88,7 +88,7 @@ type EventReference struct {
 
 // UnmarshalJSON implements json.Unmarshaller
 func (er *EventReference) UnmarshalJSON(data []byte) error {
-	var tuple []spec.RawJSON
+	var tuple []json.RawMessage
```

Comment:
I don't know why you're doing this in this PR. It's great that it's being done, but it's just adding noise to this alrady chunky PR.